README.md

```diff
@@ -72,6 +72,7 @@ New tools are being added frequently, so check this page again!
 [vale]: https://vale.sh/
 [yamlfmt]: https://github.com/google/yamlfmt
 [rustfmt]: https://rust-lang.github.io/rustfmt
+[tfsec]: https://github.com/aquasecurity/tfsec
```

Comment:
nit: we should note that we're aware of Trivy and what (If anything) we are planning to do  - someone is bound to ask